ql/src/java/org/apache/hadoop/hive/ql/parse/ReplicationSemanticAnalyzer.java

```diff
@@ -369,7 +369,6 @@ private void analyzeReplLoad(ASTNode ast) throws SemanticException {
       FileStatus[] srcs = LoadSemanticAnalyzer.matchFilesOrDir(fs, loadPath);
       if (srcs == null || (srcs.length == 0)) {
         LOG.warn("Nothing to load at {}", loadPath.toUri().toString());
```

Comment:
Can we add the last repl ID update task here itself instead of going to ReplLoadTask? Or we can get rid of this LoadSemanticAnalyzer.matchFilesOrDir call and warn message as it doesn't have any use. Also, if we remove this flow, then code flow is common for both bootstrap and incremental ReplLoadtask creation except for evDump flag.

Reply:
i think in case of bootstrap if load path is empty it should throw error ..for incremental it should just update the repl id ...anyways load path empty is a exception case ..so to keep the code simple check is added during task creation ..that will print the logs  also

Reply:
Now, we don't throw error if the load path is empty instead just log a warning and return success. I think, this scenario won't happen for bootstrap if we fail repl dump if input DB doesn't exist. So, need to fix there.